code/modules/mining/equipment/explorer_gear.dm

```diff
@@ -130,6 +130,28 @@
 	hoodtype = /obj/item/clothing/head/hooded/cloakhood/goliath
 	body_parts_covered = CHEST|GROIN|ARMS
 
+/obj/item/clothing/suit/hooded/cloak/goliath/AltClick(mob/user)
+	. = ..()
+	if(iscarbon(user))
+		var/mob/living/carbon/C = user
+		if((C.get_item_by_slot(ITEM_SLOT_NECK) == src))
+			to_chat(user, span_warning("You can't adjust [src] while wearing it!"))
+			return
+		if(C.get_item_by_slot(ITEM_SLOT_OCLOTHING == src))
+			to_chat(user, span_warning("You can't adjust [src] while wearing it!"))
+			return
+		if(!user.is_holding(src))
+			to_chat(user, span_warning("You must be holding [src] in order to adjust it!"))
+			return
+		if(slot_flags & ITEM_SLOT_OCLOTHING)
+			slot_flags = ITEM_SLOT_NECK
+			set_armor(/datum/armor/none)
+			to_chat(user,span_notice("You adjust [src] to fit around your neck."))
```

Comment:
Make these use the visible message popup thing
Also could probably mention that it removes armor on neck

Reply:
Alright, gave it visible message popup, and made you and others able to discern if it's for neck(unarmored) or overclothing(armored) via 'ceremonial use' and 'defensive use'
Also cleaned the code a bit up to make it usable for more things by using [src] instead of 'goliath cloak'